src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
The fact that the CA bundle file comes from another source/package makes no difference.  It is just a file and can still just as easily be corrupted by the same problems at would affect these files.  So it should be just as important to ensure the contents of the CA bundle file.  They both serve the same purposes.